erpnext/patches/v11_0/rename_healthcare_doctype_and_fields.py

```diff
@@ -0,0 +1,60 @@
+import frappe
+from frappe.model.rename_doc import rename_doc
+from frappe.model.utils.rename_field import rename_field
+from frappe.modules import scrub, get_doctype_module
+
+field_rename_map = {
+	"Patient Encounter": [
+		["consultation_time", "encounter_time"],
+		["consultation_date", "encounter_date"],
+		["consultation_comment", "encounter_comment"],
+		["physician", "practitioner"]
+	],
+	"Fee Validity": [
+		["physician", "practitioner"]
+	],
+	"Lab Test": [
+		["physician", "practitioner"]
+	],
+	"Patient Appointment": [
+		["physician", "practitioner"],
+		["referring_physician", "referring_practitioner"]
+	],
+	"Procedure Prescription": [
+		["physician", "practitioner"]
+	]
+}
+
+doc_rename_map = {
+	"Physician Schedule Time Slot": "Healthcare Schedule Time Slot",
+	"Physician Schedule": "Practitioner Schedule",
+	"Physician Service Unit Schedule": "Practitioner Service Unit Schedule",
+	"Consultation": "Patient Encounter",
+	"Physician": "Healthcare Practitioner"
+}
+
+def execute():
+	domain_settings = frappe.get_doc('Domain Settings')
+	active_domains = [d.domain for d in domain_settings.active_domains]
+
+	if "Healthcare" in active_domains:
```

Comment:
The patch should be run even if Healthcare domain is not activated.

Reply:
Will update as soon as possible

Reply:
Patch Updated